autogpt/configurator.py

```diff
@@ -75,11 +75,13 @@ def create_config(
 
     if gpt3only:
         logger.typewriter_log("GPT3.5 Only Mode: ", Fore.GREEN, "ENABLED")
-        CFG.set_smart_llm_model(CFG.fast_llm_model)
+        CFG.set_smart_llm_model("gpt-3.5-turbo")
```

Comment:
I might not understand why this get's hardcoded and how this is related to the chat model which was the initial scope of this?

Reply:
to me it just makes this arg open to user error, e.g. the user has set their fast_llm_model to gpt-3.5-turbo. for the arg to behave exactly as it is document i think it's best to just hardcode those values. alternatively we could log a warning if they try to run --gpt3only but don't actually have gpt3 in the env

Reply:
FWIW, I agree, but to @hdkiller's point, it's not related to the topic of the PR. It should be another PR